frontend/src/lib/test-util.ts

```diff
@@ -321,13 +321,7 @@ export function nocksAreDone(nocks: Scope[]) {
 }
 
 export async function waitForNocks(nocks: Scope[]) {
-    const timeout = options.timeout * nocks.length * 3
-    const timeoutMsg = (error: Error) => {
-        error.message = `!!!!!!!!!!! Test timed out in waitForNocks()--waited ${timeout / 1000} seconds !!!!!!!!!!!!!`
-        error.stack = ''
-        return error
-    }
-    await waitFor(() => expect(nocksAreDone(nocks)).toBeTruthy(), { timeout, onTimeout: timeoutMsg })
+    await waitFor(() => expect(nocksAreDone(nocks)).toBeTruthy())
```

Comment:
Actually I added that a while ago when I didn't understand nocks---there's a built-in timeout for every waitfor of 5 seconds. Since nocks aren't dependent on a lagging backend, there would be something wrong if even a bunch of nocks didn't finish  in 5 seconds